lib/urql.ts

```diff
@@ -7,16 +8,17 @@ import { createClient } from '@urql/core';
 const requestPolicy = 'network-only';
 
 export const nftBackedLoansClient = createClient({
-  url: process.env.NEXT_PUBLIC_NFT_BACKED_LOANS_SUBGRAPH || '',
+  url: config.nftBackedLoansSubgraph,
   requestPolicy,
 });
 
 export const eip721Client = createClient({
-  url: process.env.NEXT_PUBLIC_EIP721_SUBGRAPH || '',
+  url: config.eip721Subgraph,
   requestPolicy,
 });
 
 export const nftSalesClient = createClient({
-  url: process.env.NEXT_PUBLIC_NFT_SALES_SUBGRAPH || '',
+  // TODO: is there a better way to handle this absence?
```

Comment:
probably not, but we are probably gonna deprecate this sales subgraph soon. i am looking for a replacement API at the moment